pcsx2/gui/Panels/SpeedhacksPanel.cpp

```diff
@@ -19,45 +19,45 @@
+			m_msg_eeRate->SetForegroundColour(wxColour(L"Red"));
+			return pxEt(L"50% cyclerate. Significant reduction of CPU requirements. Speedup for very lightweight games, slows down others. FMVs and audio may stutter or skip.");
 		}
 		case -2:
 		{
-			m_msg_eecycle->SetForegroundColour(wxColour(L"Red"));
-			return pxEt(L"-2 - Reduces the EE's cyclerate to about 60%.  Moderate speedup, but may cause stuttering audio on many FMVs.");
+			m_msg_eeRate->SetForegroundColour(wxColour(L"Red"));
+			return pxEt(L"60% cyclerate. Moderate reduction of CPU requirements. Speedup for lightweight games, slows down others. FMVs and audio may stutter or skip.");
 		}
 		case -1:
 		{
-			m_msg_eecycle->SetForegroundColour(wxColour(L"Red"));
-			return pxEt(L"-1 - Reduces the EE's cyclerate to about 75%.  Mild speedup for most games with high compatibility.");
+			m_msg_eeRate->SetForegroundColour(wxColour(L"Red"));
+			return pxEt(L"75% cyclerate. Slight reduction of CPU requirements. Speedup for less demanding games, slows down others.");
 		}
 		case 0:
 		{
 			const wxColour DarkSeaGreen = wxColour(14, 158, 19);
-			m_msg_eecycle->SetForegroundColour(DarkSeaGreen);
-			return pxEt(L"0 - Default cyclerate (100%). This closely matches the actual speed of a real PS2 EmotionEngine.");
+			m_msg_eeRate->SetForegroundColour(DarkSeaGreen);
+			return pxEt(L"Default cyclerate. Runs the emulated PS2 Emotion Engine at normal speed.");
 		}
 		case 1:
 		{
-			m_msg_eecycle->SetForegroundColour(wxColour(L"Red"));
-			return pxEt(L"1 - Increases the EE's cyclerate to about 130%. Mildly increases hardware requirements, may increase in-game FPS.");
+			m_msg_eeRate->SetForegroundColour(wxColour(L"Red"));
+			return pxEt(L"130% cyclerate. Moderate increase of CPU requirements. Variable framerate games may have higher internal framerates. Moderately powerful CPU is needed to avoid slowdowns.");
 		}
 		case 2:
 		{
-			m_msg_eecycle->SetForegroundColour(wxColour(L"Red"));
-			return pxEt(L"2 - Increases the EE's cyclerate to about 180%. Increases hardware requirements, may noticeably increase in-game FPS.");
+			m_msg_eeRate->SetForegroundColour(wxColour(L"Red"));
+			return pxEt(L"180% cyclerate. Significant increase of CPU requirements. Variable framerate games will have higher internal framerates. Powerful CPU is needed to avoid slowdowns. FMVs may be slow. May cause stability problems.");
 		}
 		case 3:
 		{
-			m_msg_eecycle->SetForegroundColour(wxColour(L"Red"));
-			return pxEt(L"3 - Increases the EE's cyclerate to about 300%. Greatly increases hardware requirements, may noticeably increase in-game FPS.\nThis setting can cause games to FAIL TO BOOT.");
+			m_msg_eeRate->SetForegroundColour(wxColour(L"Red"));
+			return pxEt(L"300% cyclerate. Extreme increase of CPU requirements. Variable framerate games will have higher internal framerates. Very powerful PC CPU is needed to avoid slowdowns. FMVs may be slow. May cause stability problems.");
```

Comment:
`Extreme increase of CPU requirements.`
`Very powerful PC CPU is needed to avoid slowdowns.`

I still find it a bit unnecessary to mention that a powerful CPU is needed since IMO the first sentence covers that. Same applies to the other EE overclock messages.